src/EnergyPlus/WeatherManager.cc

```diff
@@ -8657,7 +8650,7 @@ namespace Weather {
         }
     }
 
-    void ForAllHrTs(EnergyPlusData &state, std::function<void(int, int)> f)
+    void ForAllHrTs(EnergyPlusData const &state, std::function<void(int, int)> f)
```

Comment:
[src/EnergyPlus/WeatherManager.cc:8660]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const